packages/frontend/src/components/common/Filters/FilterInputs/DefaultFilterInputs.tsx

```diff
@@ -23,6 +23,73 @@ export type FilterInputsProps<T extends ConditionalRule> = {
     disabled?: boolean;
 };
 
+const getPlaceholderByFilterTypeAndOperator = (
+    filterType: FilterType,
+    filterOperator: FilterOperator,
+) => {
+    switch (filterType) {
+        case FilterType.NUMBER:
+            switch (filterOperator) {
+                case FilterOperator.EQUALS:
+                case FilterOperator.NOT_EQUALS:
+                case FilterOperator.LESS_THAN:
```

Comment:
This output doesn't match the spect, less_than should be `Enter value`

Reply:
![image](https://github.com/lightdash/lightdash/assets/1983672/1dde6871-6f0a-408a-818c-6d7d649bf305)

Reply:
Thanks for checking! changed it and added screen recording ^